<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leSTLFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSTLFileOpen">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>파일열기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConvert">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변환</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSend">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShowOriginal">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>원본 보기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShowConvert">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변환된 형태 보기</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>메이플스토리 OTF</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
